docs/developer/unsupported_server_args.md

```diff
@@ -0,0 +1,76 @@
+<h1 align="center">Unsupported server args override for Velero deployment and Node-agent Daemon set containers</h1>
+
```

Comment:
Write a paragraph explaining the purpose of this feature, its impact on the current configuration, and include a link to the design architecture document. This feature is primarily intended for developers and special cases, allowing for the complete override of all arguments. Note needs to be included that since this feature overrides all existing arguments, any previously specified options must be repeated in the override.